rpm/amber-web-authorization.spec

```diff
@@ -0,0 +1,108 @@
+Name: libamberwebauthorization
```

Comment:
It's better to name the main package in the same way as the repository to prevent confusion.
```suggestion
Name: amber-web-authorization
```
Then you can explicitly name only one sub-package with the main library as `libamberwebauthorization` which makes this spec simpler and more logical.

Reply:
:+1:

Reply:
done